### ISSUE TITLE
Default cops that I wouldn't really want

### ISSUE THREAD

**Issue by swanson** [standardrb/standard-rails#1]
It's Sunday night, so what better to do than review the `rubocop-rails` defaults! I know there is a bigger discussion happening at RailsConf but I will not be in attendance so wanted to chime in.

Overall, I think the vast majority of cops are reasonable, helpful, and should be easy wins. The following were the ones that made me say "really?" or "ugh I would be annoyed to have that auto-corrected on my code"

---

[`Rails/ActiveSupportAliases`](https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsactivesupportaliases) 
Summary: "Checks that ActiveSupport aliases to core ruby methods are not used." Enforces `start_with?` `<<` and `unshift` over `starts_with?` `append` `prepend`

Comments: This seems worse as the AS aliases are more clear and readable. I wouldn't ever tell someone to switch `prepend` to `unshift`.

---

[`Rails/AssertNot`](https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsassertnot)
Summary: "Use assert_not instead of assert !"

[`Rails/RefuteMethods`](https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsrefutemethods)
Summary: "Use assert_not methods instead of refute methods."

Comments: Seems over-reaching a bit as it is about test styling. Tests may read better with different styles "Assert that this record is not completed -> `assert !@record.completed?` vs `assert_not @record.completed?`

Should this be part of a hypothetical `standard-minitest`? 

---

[`Rails/SkipsModelValidations`](https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsskipsmodelvalidations)
Summary: disallow methods that skip AR validations

Comments: Too spicy. Plenty of valid use-cases in context for `update_all`, `touch`, or `update_column` -- usually for skipping callbacks. If it would be a net benefit, maybe consider the allowing `touch` option.

**Comment by MatheusRich** [comment]
Aside from the minitest ones (which I don't use so I have no opinion), I completely agree with @swanson.

`unshift` is a terrible method name and always confused me. `prepend` is short and clear.

**Comment by searls** [comment]
> Aside from the minitest ones (which I don't use so I have no opinion), I completely agree with @swanson.
> 
> `unshift` is a terrible method name and always confused me. `prepend` is short and clear.

Yes, fear not, this is one we voted on at RailsConf (and if I'm misremembering, it's one we plan to leave off anyway) for the reasons cited above. Seeing how many AS aliases eventually made their way into Ruby's stdlib, it'd be a little silly to ban them as unhealthy.

Going to close this, but hope to get over the hump on a 1.0 release that specifies an initial slate of rules with a cleaned up YAML file to scrutinize
